### ISSUE TITLE
aperture size in % is twice as big as I'd expect

### ISSUE THREAD

**Issue by tpronk** [u01ai11/MouseView.js#14]
Hey again :),

I ran a test where I used these settings:
```
mouseview.params.apertureSize = '50%';
mouseview.params.apertureGauss = 0;
```

I noticed that the aperture perfectly filled the height of my window, but I would expect it to be half the height.

Also, noticed in the code that the aperture scaling is relative to `window.innerHeight`, but that could give unexpected behavior with screens in portrait orientation.

**Comment by u01ai11** [comment]
Thanks for spotting all these things -- I'll have a look, and try and work out why this is. 

The percentage scaling was done quickly to give users an option other than pixels. I have a plan to use some kind of ratio (H:W) scaling (kind of like how UI libraries deal with screen sizes and scaling) to solve these issues with mobile devices and unusual orientations.

**Comment by u01ai11** [comment]
Ahh, this is a silly radius/circumference mix up. Solved by dividing by two :)

**Comment by u01ai11** [comment]
p.s. this was fixed in 7b0baeffb15dc2bb85caddc7a32e9e45ad9702c3 Thanks
